src/controller/CHIPDeviceController.h

```diff
@@ -640,6 +640,14 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
      */
     CHIP_ERROR ValidateAttestationInfo(const Credentials::DeviceAttestationVerifier::AttestationInfo & info);
 
+    /**
+     * @brief
+     *   This function validates the revocation status of the DAC Chain sent by the device.
+     *
+     * @param[in] info Structure contatining all the required information for validating the device attestation.
+     */
+    CHIP_ERROR CheckForRevokedDACChain(const Credentials::DeviceAttestationVerifier::AttestationInfo & info);
```

Comment:
Why is this public API?

Reply:
Just followed the existing conventions and made it public. Note that ValidateAttestationInformation is also public.

Reply:
As discussed on Slack: it shouldn't be: it will mess with the state machine in a way that makes it unsafe to call from outside this class.  So does the new function.  So it should not be public.

Reply:
Addressed in latest commit.